lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -216,7 +213,7 @@ def support_bulk_alter?
             true
           when POSTGRESQL
             # Add bulk alter support for PostgreSQL in 5.2.0
-            # @see https://github.com/rails/rails/pull/31331
+            # https://github.com/rails/rails/pull/31331
```

Comment:
[memo] Remove only the @see tag since it is not output in the reference but the @see tag is meaningless here

Reply:
```suggestion
            # See: https://github.com/rails/rails/pull/31331
```